src/app/modules/project/modal-highlight-delete/modal-highlight-delete.component.html

```diff
@@ -39,9 +39,10 @@ <h2>Delete highlight</h2>
                 <div class="highlight-wrapper">
                   <input type="checkbox" id="checkbox" [formControlName]="highlight.id" class="" />
                   <label for="checkbox">
-                    <div class="row no-gutters">
-                      <div class="col-6"><strong>Start date:</strong>{{highlight.startDate}}</div>
-                      <div class="col-6"><strong>End date:</strong>{{highlight.endDate}}</div>
+                    <div class="row">
+                      <div class="col-4"><strong>Start date:</strong>{{highlight.startDate}}</div>
+                      <div class="col-4"><strong>End date:</strong>{{highlight.endDate}}</div>
+                      <div class="col-4"><strong>Description:</strong>{{highlight.description}}</div>
```

Comment:
Let's revert this change, after looking at the modal I don't think this will really give the user more information. It will make the modal look messy (especially with longer descriptions) and will not serve much of a purpose at this moment.

Reply:
This file was not reverted properly.  Please check the file history and revert back to the correct version.